CMakeLists.txt

```diff
@@ -7,19 +7,6 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 set(LINK_LIBS "")
 
-# Use the package PkgConfig to detect GTK+ headers/library files
-find_package(PkgConfig REQUIRED)
-pkg_check_modules(GTK3 REQUIRED gtk+-3.0)
```

Comment:
For some reason, this snuck through our last PR -- might have been a merge conflict of some kind.